Cargo.toml

```diff
@@ -6,6 +6,7 @@ members = [
 
 [profile.release]
 lto = true
+strip = true
```

Comment:
Is this required? This is new to me

Reply:
It's not required, but it will strip out debug information and result in smaller binaries for release builds.